pulp_docker/tests/functional/api/test_tagging_images.py

```diff
@@ -46,24 +46,28 @@ def test_01_tag_first_image(self):
         This test checks if the tag was created in a new repository version.
         """
         manifest_a = self.get_manifest_by_tag('manifest_a')
-        response = self.tag_image(manifest_a, 'new_tag')
+        self.tag_image(manifest_a, 'new_tag')
 
         new_repository_version_href = '{repository_href}versions/{new_version}/'.format(
             repository_href=self.repository['_href'],
             new_version='2'
         )
 
-        created_tag = self.client.get('{unit_path}?{filters}'.format(
+        added_tags_href = '{unit_path}?{filters}'.format(
             unit_path=DOCKER_TAG_PATH,
             filters=f'repository_version_added={new_repository_version_href}'
-        ))['results'][0]
+        )
+        created_tag = self.client.get(added_tags_href)['results'][0]
         self.assertEqual(created_tag['name'], 'new_tag', created_tag['name'])
 
-        self.assertEqual(
-            set(response['created_resources']),
-            {new_repository_version_href, created_tag['_href']},
-            set(response['created_resources'])
-        )
+        repository_version = self.client.get(new_repository_version_href)
+
+        added_content = repository_version['content_summary']['added']
+        added_tags = added_content['docker.tag']['href']
+        self.assertEqual(added_tags, added_tags_href, added_tags)
```

Comment:
AFAICT, these assertions just show that these are all the same href (added_tags is in here twice). This doesn't tell us much. Instead, I suggest we look at the content counts. Similar to:
https://github.com/pulp/pulp_docker/blob/master/pulp_docker/tests/functional/api/test_recursive_add.py#L104